Simplenote/SplitViewController.swift

```diff
@@ -0,0 +1,164 @@
+import Foundation
+
+
+// MARK: - SplitViewController
+//
+@objc
+class SplitViewController: NSSplitViewController {
+
+    /// Indicates if the Notes List is collapsed
+    ///
+    var isFocusModeEnabled: Bool {
+        splitViewItem(ofKind: .notes).isCollapsed
+    }
+
+
+    // MARK: - Overridden Methods
+
+    override func loadView() {
+        let splitView = SplitView()
+        splitView.isVertical = true
+        self.splitView = splitView
+        self.view = splitView
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        splitView.autosaveName = "Please Save Me!"
```

Comment:
😆 I had to google to see what this was about.  Is that from [this](https://stackoverflow.com/questions/16587058/nssplitview-auto-saving-divider-positions-doesnt-work-with-auto-layout-enable)?  Should we add a citation, or maybe leave a comment about the glitch?

Reply:
Wow!! Actually, I was aware of this problem (divider positions not being saved). We had a really ugly fix, in the past, which is being removed in this very same PR.

The string matching is **an astronomical coincidence** (I was listening to an obscure song when this was written!!).

I'll add a comment, thank you!!!